test/controllers/users_controller_test.rb

```diff
@@ -32,4 +32,41 @@
       expect(flash[:result_text]).must_equal "User not found!"
     end
   end
+
+  describe "login" do
```

Comment:
this is testing your "create" action not your login action - maybe either change the name of this test?